### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ElminD**
I used Grammarly and other tools to update the file

docs: Updated grammar Contributing
Signed-off-by: Elmin Didic [edidic@iastate.edu]

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -81,15 +81,15 @@ The DCO is a declaration attached to every contribution made by every developer
 
 **Each commit must include a DCO which looks like this:**
 
-```txt
+"`txt
```
Comment (thecodrr) [inline CONTRIBUTING.md:84]: We are creating a codeblock here. The requested change here is incorrect.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -81,15 +81,15 @@ The DCO is a declaration attached to every contribution made by every developer
...
 Signed-off-by: Jane Smith <jane.smith@email.com>
 ```
 
 On VSCode, it looks like this:
 
 ![](resources/vscode-signed-off-commit.png)
 
-You may type this line on your own when writing your commit messages. However, if your [user.name](http://user.name) and [user.email](http://user.email) are set in your git configs, you can use `-s` or `--signoff` to add the `Signed-off-by` line to the end of the commit message automatically.
+You may type this line on your own when writing your commit messages. However, if your [user.name](http://user.name) and [user.email](http://user.email) are set in your git configs, you can use `-s' or `--signoff` to add the `Signed-off-by` line to the end of the commit message automatically.
```
Comment (thecodrr) [inline CONTRIBUTING.md:92]: The requested change here is incorrect.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -119,7 +119,7 @@ All commits must include valid scopes in the commit message. **Valid commit scop
 
 **Some example commits would look like this:**
 
-```txt
+"`txt
```
Comment (thecodrr) [inline CONTRIBUTING.md:122]: The requested change here is incorrect.

**Review by thecodrr — CHANGES_REQUESTED** [review]
I appreciate you taking the time to pass this through Grammarly & friends. There are some mistakes, however, probably due to this being Markdown which Grammarly doesn't work well with.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ElminD) [reply CONTRIBUTING.md]: Fixed

**Comment by ElminD** [comment]
Should be all fixed

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -4,13 +4,13 @@ Thank you so much for considering to contribute to Notesnook! If you have no ide
 
 ## Ways to Contribute
 
-### Reporting a bug
+### Reporting a bug
```
Comment (thecodrr) [inline CONTRIBUTING.md:7]:
```suggestion
### Reporting a bug
```

**Pull request merged**